zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -574,6 +574,13 @@ where
         .map_err(Error::from)
 }
 
+type StepResult<DbT> = (
```

Comment:
nit: this would be nicer as a struct

Reply:
You may be right. Shall I make it one in #1468?